test-scripts/0.8.0/docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
       - PEM111_SOAP_VALIDATION_ENABLED
       - PEM111_ITK_ODS_CODE_LIST
       - PEM111_ITK_DOS_ID_LIST
+      - PEM111_VERSION
```

Comment:
version 0.8.0 has already been release, there's no need to add anything here
We will include it in the next release